crates/openapi/src/routes/merchant_connector_account.rs

```diff
@@ -139,7 +139,7 @@ pub async fn connector_create() {}
     ),
     tag = "Merchant Connector Account",
     operation_id = "Retrieve a Merchant Connector",
-    security(("admin_api_key" = []))
+    security(("api_key" = []))
```

Comment:
we can revert these, because these will authenticated by jwt or admin_api_key, if this in not intended do the revert below also

Reply:
connector retrieve, list and update no longer use Admin api key. They use merchant api key. 
This change was done in this PR https://github.com/juspay/hyperswitch/pull/5572/files